### ISSUE TITLE
convertPathData does not apply transform to stroke-dasharray

### ISSUE THREAD

**Issue by JoKalliauer** [svg/svgo#1053]
Prozessing [File:Kalliauer_et_al_2018_Acta_Mechanica_Fig11.svg](https://commons.wikimedia.org/wiki/File:Kalliauer_et_al_2018_Acta_Mechanica_Fig11.svg) leads to a change of the `stroke-dasharray`-interval:

>  svgo -i [input.svg.txt](https://github.com/svg/svgo/files/2476701/input.svg.txt) -o [output.svg.txt](https://github.com/svg/svgo/files/2476702/output.svg.txt) --pretty --indent=1

# Input
[input.svg.txt](https://github.com/svg/svgo/files/2476701/input.svg.txt)
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 265.34 193.34">
 <path fill="none" stroke="#000" stroke-dasharray="4.52,4.52" stroke-linejoin="bevel" stroke-width="1.51" d="M233.27 356.92l8.533 2.378 8.513 2.382 8.428 2.398 8.282 2.484 8.145 2.597 7.98 2.748 7.775 2.98 7.571 3.076 7.344 3.234 7.162 3.444 6.869 3.739 6.704 3.993 6.364 4.311 5.735 3.892 5.156 3.385 4.598 3.504 4.517 3.679 4.055 4.249 3.594 4.927 3.349 5.378 3.006 5.576 2.684 5.472 2.395 5.342 2.065 4.874 1.833 4.077 1.429 3.412 1.141 2.981 1.128 2.372.715 2.356.696 1.873.533 1.617.411 1.387.317 1.078.355.772.084.704 1.057.028M242.89 463.55h22.678" transform="matrix(1.3333 0 0 -1.3333 -263.83 640.79)"/>
</svg>
```
![input](https://user-images.githubusercontent.com/15687581/46920525-ca1f9180-cfef-11e8-9bcc-004ee72a50ff.png)

# Output
[output.svg.txt](https://github.com/svg/svgo/files/2476702/output.svg.txt)
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 265.34 193.34">
 <path fill="none" stroke="#000" stroke-dasharray="4.52,4.52" stroke-linejoin="bevel" stroke-width="2.013283" d="M47.189 164.909l11.377-3.171 11.35-3.176 11.237-3.197 11.043-3.312 10.86-3.463 10.64-3.664 10.366-3.973 10.094-4.101 9.792-4.312 9.549-4.592 9.158-4.985 8.939-5.324 8.485-5.748 7.646-5.189 6.875-4.513 6.13-4.672 6.023-4.905 5.406-5.665 4.792-6.57 4.466-7.17 4.007-7.435 3.579-7.295 3.193-7.123 2.754-6.498 2.443-5.436 1.906-4.55 1.521-3.974 1.504-3.163.953-3.14.928-2.498.71-2.156.549-1.85.423-1.437.473-1.029.112-.939 1.41-.037M60.014 22.739h30.237"/>
</svg>
```
![output](https://user-images.githubusercontent.com/15687581/46920527-cc81eb80-cfef-11e8-986e-7f44ab240b1c.png)

<details>

<summary> Copyright </summary>

CC-BY 4.0

Johannes Kalliauer in Acta Mechanika

https://commons.wikimedia.org/wiki/File:Kalliauer_et_al_2018_Acta_Mechanica_Fig11.svg

</details>

**Comment by TrySound** [comment]
Fixed in https://github.com/svg/svgo/commit/dd37fcfebbe2f3cea687ad0bf890ff83ab3c9eb7
Will release soon

**Comment by TrySound** [comment]
Fixed in 2.2.0
